starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -14,5 +14,14 @@
   <label for="contact-image"></label>
   <input type="text" [(ngModel)]="newContact.image">
 </div>
+<button (click)="addContact()">Submit New Contact</button>
+<ul>
+  <li *ngFor="let contact of contacts">
+   <p>{{contact.name}}</p>
+   <p>{{contact.email}}</p> 
+   <p> {{contact.phoneNumber}} </p>    
+    <img [src]= "contact.image" alt="LOADING">
```

Comment:
It is best practice to always include descriptive alt text (if the image is not purely decorative), e.g.: ```alt="Profile picture of {{ contact.name }}"```